packages/@sanity/schema/src/legacy/Schema.ts

```diff
@@ -42,22 +56,33 @@ function compileRegistry(schemaDef: any) {
  * @beta
  */
 export class Schema {
-  _original: {name: string; types: any[]}
+  _original: {name: string; types: any[]; parent?: Schema}
   _registry: {[typeName: string]: any}
+  _localTypeNames: string[]
```

Comment:
Change to `#localTypeNames: string[]` to get proper privacy?

(It's very tempting to do that for the other "private" properties here too, but god knows who might be relying on them already)

Reply:
Agreed! Fixed in a follow-up commit!